package.json

```diff
@@ -47,6 +47,7 @@
     "moment-timezone": "0.5.14",
     "mycrypto-eth-exists": "1.0.0",
     "mycrypto-shepherd": "1.4.0",
+    "next": "8.1.0",
```

Comment:
I couldnt find where this was used. Any insight?

Reply:
removed